docs/snapcraft-parts.md

```diff
@@ -164,3 +164,14 @@ If you only need to embed a Java runtime, add a part with the jdk type. This
 will pull a relocatable OpenJDK via the default-jdk Ubuntu package and will
 set the proper `JAVA_HOME` and `PATH` environment variables in wrapper
 scripts generated by snapcraft or when running the app locally.
+
+
+## Reusing parts
+
+Successful snapcraft parts can be published in
+[the wiki](https://wiki.ubuntu.com/Snappy/Parts) and reused by other snapcraft.yaml
+files.  A part that does not include the 'plugin' option will be searched for on
+[the wiki](https://wiki.ubuntu.com/Snappy/Parts) and the content from there will be
+used to build the part locally.  As the number of published parts increases writing
```

Comment:
I think you are missing a comma:
As the number of published parts increases, writing